main.go

```diff
@@ -269,13 +302,10 @@ func tsFieldToTimestamp(input interface{}) (*time.Time, error) {
 	return &zeroTime, fmt.Errorf("don't know how to turn %T (value %s) into a time.Time object", input, input)
 }
 
+// Using the log fields of stack driver: https://cloud.google.com/logging/docs/structured-logging
 func (p *processor) maybePrettyPrintZapdriverLine(line string, lineData map[string]interface{}) (string, error) {
-	timeField := "time"
+	timeField := "timestamp"
 	timeValue := lineData[timeField]
-	if lineData[timeField] == nil {
-		timeField = "timestamp"
-		timeValue = lineData[timeField]
-	}
```

Comment:
Ok what's that change exactly? Tests are probably failing straight with this change.

 The zapdriver had it's own format, I don't think we should change it. I see two options, those changes are reverted or you add a new method `maybePrettyPrintGCPCloudLoggingLine(...)` that deals with correct format.